http/server.lua

```diff
@@ -746,6 +773,11 @@ local function process_client(self, s, peer)
     while true do
         local hdrs = ''
 
+        connections[fiber.id] = fiber.clock()
```

Comment:
"fiber.id" is a function. Not just numeric value. Probably it should be `fiber.id()`, anyway how does it pass tests?

Reply:
```lua
--- a/http/server.lua
+++ b/http/server.lua
@@ -774,7 +774,8 @@ local function process_client(self, s, peer)
         local logger = get_request_logger(self.options)
 
         if self.options.idle_timeout > 0 then
-            connections[fiber.id] = fiber.clock()
+            local id = fiber.id()
+            connections[id] = fiber.clock()
             close_idle_connections(self.connections, self.options.idle_timeout, logger)
         end
 
```

>  anyway how does it pass tests?

I don't know a reliable way to test this feature and tests are simple right now.

Reply:
In fact, I need a fiber object, not a fiber id. Because `close_idle_connections()` should somehow call `fiber:cancel()`.